### PR TITLE
Ghg warpath this time for real

### PR DESCRIPTION
**Pull request by Fraggy1407**
Everything fixed up and bypasses added
The "outright go to war" path was removed for now

### CODE REVIEW

File: TGC/decisions/Germany.txt [hunk 1]
```diff
@@ -4109,7 +3946,6 @@ political_decisions = {
 				AUS = { exists = yes in_sphere = THIS }
 				KUK = { exists = yes in_sphere = THIS }
 			}
-			has_country_flag = brothers_war_winner
```
Comment (anzu-3278) [inline TGC/decisions/Germany.txt:4112]: Is this related to the GHG?

File: TGC/decisions/MEX.txt [hunk 2]
```diff
@@ -1354,8 +1354,8 @@ political_decisions = {
 	glory_in_battle_blood_in_the_field = { #madebyFinnBroandAndJPADUN
 		potential = {
 			tag = MEX
-			war_with = USA
-			NOT = { year = 1850 }
```
Comment (anzu-3278) [inline TGC/decisions/MEX.txt]: Looks like you picked up some parallel changes again.

File: TGC/decisions/TKS.txt [hunk 3]
```diff
@@ -371,6 +371,7 @@ political_decisions = {
 						primary_culture = kirgiz
 						primary_culture = tajik
 						primary_culture = kazak
+						primary_culture = uighur
```
Comment (anzu-3278) [inline TGC/decisions/TKS.txt]: Whoops.

File: TGC/localisation/00_events_1000000000.csv [hunk 4]
```diff
@@ -13,8 +13,8 @@ EVTOPTA1904191419;An Atlantic connection would greatly boost our trade!;;;;;;;;;
 EVTOPTB1904191419;I don't think we'll ever get that road.;;;;;;;;;;;;;x
 EVTDESC1904191420;Sell Fort Ross to the Mexican citizen John Sutter?;;;;;;;;;;;;;x
 EVTDESC1904191421;$FROMCOUNTRY$ has agreed to sell the Fort Ross trading post to us! Long live $COUNTRY$!;;;;;;;;;;;;;x
-EVTDESC1904191422;The British have come forth to us with a proposal to settle the border between our state of Yucatan and their colony of Belize. In order to prevent future conflict and the native encroachment on our respective territories, they propose that we set the border at the Rio Hondo river. While this would mean we would be renouncing some territorial claims, this treaty would also help us bring the problem of the rebellious Maya in Yucatan under control.;;;;;;;;;;;;;x
-EVTNAME1904191422;The Belize-Yucatan Border;;;;;;;;;;;;;x
+EVTNAME1904191422;The British demand full sovereignity over Belize!;;;;;;;;;;;;;x
```
Comment (anzu-3278) [inline TGC/localisation/00_events_1000000000.csv]: I think I'll start having ptsd from seeing this event's localization come back from the dead

File: TGC/decisions/Germany.txt (hunk 1 above)
Comment (Fraggy1407) [reply TGC/decisions/Germany.txt:4112]: No this is part of what obstination brought to my attention related to the inclusion of austria into germany post three hurrahs.

File: TGC/decisions/MEX.txt (hunk 2 above)
Comment (Fraggy1407) [reply TGC/decisions/MEX.txt]: yea i will merge master in again

File: TGC/decisions/TKS.txt (hunk 3 above)
Comment (Fraggy1407) [reply TGC/decisions/TKS.txt]: ?

File: TGC/localisation/00_events_1000000000.csv (hunk 4 above)
Comment (Fraggy1407) [reply TGC/localisation/00_events_1000000000.csv]: yeaaaa that happened again

File: TGC/decisions/MEX.txt (hunk 2 above)
Comment (anzu-3278) [reply TGC/decisions/MEX.txt]: This still exists

File: TGC/decisions/TKS.txt (hunk 3 above)
Comment (anzu-3278) [reply TGC/decisions/TKS.txt]: And this

File: TGC/decisions/MEX.txt (hunk 2 above)
Comment (Fraggy1407) [reply TGC/decisions/MEX.txt]: resolved

File: TGC/decisions/TKS.txt (hunk 3 above)
Comment (Fraggy1407) [reply TGC/decisions/TKS.txt]: resolved

File: TGC/localisation/00_events_1000000000.csv (hunk 4 above)
Comment (Fraggy1407) [reply TGC/localisation/00_events_1000000000.csv]: resolved
